.github/workflows/build-arch.yml

```diff
@@ -0,0 +1,67 @@
+# This workflow builds the AYAB package for Arch Linux
+# and stores it as an artifact
+
+name: build-arch
+
+on:
+  push:
+    branches:
+      - master
+      # - '*-dev'
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```

Comment:
Should be running with ubuntu-latest as well, shouldn't it?

Reply:
For the time being I am keeping the Linux scripts on Ubuntu 16.04 for the sake of compatibility as this is still a supported version. Anything that works on 16.04 should also work on 18.04, but the reverse is not necessarily true.